assets/components/molecules/card/card.scss

```diff
@@ -8,6 +8,14 @@
   margin-bottom: $grid-gutter-width;
 }
 
+a.card.bg-gray-100 {
```

Comment:
This is too specific and might not have the result we want with cards outside of this component. I guess a class would be better in that case.